dataRetriever/factory/metachain/resolversContainerFactory.go

```diff
@@ -456,8 +456,7 @@ func (rcf *resolversContainerFactory) IsInterfaceNil() bool {
 func (rcf *resolversContainerFactory) generateTrieNodesResolver() ([]string, []dataRetriever.Resolver, error) {
 	shardC := rcf.shardCoordinator
 
-	identifierTrieNodes := factory.TrieNodesTopic + shardC.CommunicationIdentifier(sharding.MetachainShardId)
-
+	identifierTrieNodes := factory.AccountTrieNodesTopic + shardC.CommunicationIdentifier(sharding.MetachainShardId)
```

Comment:
Is a resolver for `factory.ValidatorTrieNodesTopic` needed as well?

Reply:
this was done in a proper way on development, will merge the code and will disappear.